client/Components/FormContainer.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+import SelfServeOptions from './SelfServeOptions';
+import GetInTouchForm from './GetInTouchForm';
+import BookATourForm from './BookATourForm';
+
+const FormContainer = () => {
+
+  const [ formType, setFormType ] = useState('self-serve');
+
+  const handleFormChange = (formType) => {
+    setFormType(formType);
+  }
+
+  let form;
```

Comment:
Open to cleaner ways of handling this conditional rendering logic

Reply:
Could try also passing formType and checking the if statement in the component itself, or put them in an object and index it by formType